### PR TITLE
Timeout increase

### PR DESCRIPTION
**Pull request by djodjo3**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Increasing the traces timeout during Docker build, in order to avoid deployment failures
| Type?             | Improvement, mostly 
| BC breaks?        | Nope
| Deprecations?     | Nope
| Fixed ticket?     | None, to my knowledge
| How to test?      | Just build the Docker image to make sure it does not fail us
| Possible impacts? | None that I'm aware of

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ COPY --from=builder1 /app /app
 
 RUN composer create-project prestashop/traces
 
-RUN ./traces/traces -u $ENV_USER_LOGIN -p $ENV_USER_PASS -o PrestaShop --config="./traces/config.dist.yml"
+RUN ./traces/traces -u $ENV_USER_LOGIN -p $ENV_USER_PASS -o PrestaShop --config="./traces/config.dist.yml" --timeout 30000
```
Comment (Progi1984) [inline Dockerfile:24]:
```suggestion
RUN ./traces/traces -u $ENV_USER_LOGIN -p $ENV_USER_PASS -o PrestaShop --config="./traces/config.dist.yml" --timeout 30
```

The timeout is in seconds, not milliseconds.

**Review by NeOMakinG — DISMISSED** [review]
Please, rebase your branch into develop

File: Dockerfile (hunk 1 above)
Comment (djodjo3) [reply Dockerfile:24]: Oh ! That's odd, error message was in ms:
```cURL error 28: Operation timed out after 2000 milliseconds```

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Could you please rebase your pull request, a lot of commits has been added for nothing :sweat_smile:

**Comment by djodjo3** [comment]
Hello,
Did some stuff, seems to do the job.
Let me know if there's anything else ;)

File: Dockerfile [hunk 2]
```diff
@@ -21,7 +21,7 @@ COPY --from=builder1 /app /app
 
 RUN composer create-project prestashop/traces
 
-RUN ./traces/traces -u $ENV_USER_LOGIN -p $ENV_USER_PASS -o PrestaShop --config="./traces/config.dist.yml"
+RUN ./traces/traces -u $ENV_USER_LOGIN -p $ENV_USER_PASS -o PrestaShop --config="./traces/config.dist.yml"  --timeout 30
```
Comment (PierreRambaud) [inline Dockerfile:24]:
```suggestion
RUN ./traces/traces -u $ENV_USER_LOGIN -p $ENV_USER_PASS -o PrestaShop --config="./traces/config.dist.yml" --timeout 30
```

**Review by PierreRambaud — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Pull request merged**